helpers/lib/cdnify.js

```diff
@@ -26,28 +24,38 @@ module.exports = globals => {
             return '';
         }
 
-        if (/^(?:https?:)?\/\//.test(path)) {
+        if (path.startsWith('http') || path.startsWith('//')) {
             return path;
         }
-
+        //this block is to strip leading . and / of path
```

Comment:
Minor formatting and word change request: `// This block is to strip leading . and / from path`

Reply:
done.